content/sensu-go/5.4/reference/checks.md

```diff
@@ -93,7 +72,221 @@ At every execution of a check command – regardless of success or failure – t
+For checks with `round_robin` set to `true`, Sensu executes the check once per interval, cycling through the available agents alphabetically according to agent name.
+
+For example, for three agents configured with the `system` subscription (agents A, B, and C), a check configured with the `system` subscription and `round_robin` set to `true` results in one monitoring event per interval, with the agent creating the event following the pattern A -> B -> C -> A -> B -> C for the first six intervals.
+
+_PRO TIP: You can use round-robin to distribute check execution workload across multiple agents when using [proxy checks with the proxy requests attributes](#using-a-proxy-check-to-monitor-a-multiple-proxy-entities)._
+
+### Scheduling
+
+You can schedule checks using the `interval`, `cron`, and `publish` attributes.
+Sensu requires that checks include either an `interval` attribute (interval scheduling), a `cron` attribute (cron scheduling), or the `publish` attribute set to `false` (ad-hoc scheduling).
+
+| check attributes | interval scheduling | cron scheduling | ad-hoc scheduling
+| --- | --- | --- | --- |
+| `interval` | ✅ | ❌ | ❌
+| `cron` | ❌ | ✅ | ❌
+| `publish` | `true` | `true` | `false`
+
+#### Interval scheduling
+
+You can schedule a check to be executed at regular intervals using the `interval` and `publish` check attributes.
+For example, to schedule a check to execute every 60 seconds, set the `interval` attribute to `60` and the `publish` attribute to `true`.
+
+_NOTE: When creating an interval check, Sensu calculates an initial offset to splay the check's
+first scheduled request.
+This helps to balance the load of both the backend and the agent, and may result in a delay before initial check execution._
+
+**Example interval check**
+
+{{< highlight json >}}
+{
+  "type": "CheckConfig",
+  "api_version": "core/v2",
+  "metadata": {
+    "name": "interval_check",
+    "namespace": "default"
+  },
+  "spec": {
+    "command": "check-cpu.sh -w 75 -c 90",
+    "subscriptions": ["system"],
+    "handlers": ["slack"],
+    "interval": 60,
+    "publish": true
+  }
+}
+{{< /highlight >}}
+
+#### Cron scheduling
+
+You can also schedule checks using [cron syntax][14].
+For example, to schedule a check to execute once a minute at the start of the minute, set the `cron` attribute to `* * * * *` and the `publish` attribute to `true`.
+
+**Example cron check**
+
+{{< highlight json >}}
+{
+  "type": "CheckConfig",
+  "api_version": "core/v2",
+  "metadata": {
+    "name": "cron_check",
+    "namespace": "default"
+  },
+  "spec": {
+    "command": "check-cpu.sh -w 75 -c 90",
+    "subscriptions": ["system"],
+    "handlers": ["slack"],
+    "cron": "* * * * *",
+    "publish": true
+  }
+}
+{{< /highlight >}}
+
+#### Ad-hoc scheduling
+
+In addition to automatic execution, you can create checks to be scheduled manually using the [checks API](../../api/check#the-checkscheckexecute-api-endpoint).
```

Comment:
```suggestion
In addition to automatic execution, you can create checks to be scheduled manually using the [checks API](../../api/checks/#the-checkscheckexecute-api-endpoint).
```

I think:

https://docs.sensu.io/sensu-go/5.4/api/checks/#the-checkscheckexecute-api-endpoint

Reply:
Great catch! Fixed in https://github.com/sensu/sensu-docs/pull/1339/commits/b4718fd8f78cff3ae8ea24f7bc98f1c6d617c4dc